pkgs/by-name/op/opensoldat/package.nix

```diff
@@ -84,13 +94,18 @@ stdenv.mkDerivation rec {
   # TODO(@sternenseemann): set proper rpath via cmake, so we don't need autoPatchelfHook
   runtimeDependencies = [ xorg.libX11 ];
 
+  patches = [
+    ./cmake_min_req.patch
+    ./build_gns.patch
```

Comment:
Can you submit these patches as PRs upstream and link them here as well?

Reply:
Done, good call.